choreolib/src/test/java/choreo/ChoreoTests.java

```diff
@@ -56,10 +54,10 @@ public class ChoreoTests {
 }
 """;
 
-  private static final Gson GSON =
-      new GsonBuilder()
-          .registerTypeAdapter(EventMarker.class, new EventMarker.Deserializer())
-          .create();
+  // private static final Gson GSON =
+  //     new GsonBuilder()
+  //         .registerTypeAdapter(EventMarker.class, new EventMarker.Deserializer())
+  //         .create();
```

Comment:
What's this commenting for?

Reply:
It was unused. Idk why its unused now vs b4 tbh, just commenting it for now